src/workers/forum_automod.py

```diff
@@ -12,6 +12,10 @@
 logger = logging.getLogger(__name__)
 
 
+class UnableToDM(Exception):
+    pass
+
+
 @tasks.loop(hours=24)
```

Comment:
I think this should be at a specified time, rather than just once every 24 hours. Not blocking for now, we can revisit later

Reply:
I'll make an issue on the board for that and figure it out down the line👍